### DISCUSSION TITLE
Enforce user single session across devices/browsers

### DISCUSSION THREAD

**Discussion by zaic20** [OpenIDC/mod_auth_openidc#841] [Q&A]
Hi All,

is it possible to enforce a single session across devices (Browser) in someway? The requirement is to invalidate the mod_auth_openidc session active on an another browser/device if the user starts a new session. 

I cannot find anything related to this.

Thanks so much

**Comment by zandbelt** [comment]
no that's not possible

**Reply by zaic20** [reply]
@zandbelt after 1 year, do you have any news on this? there is any plan to implement this control? In case do you have any suggestions on how to enforce single user session across browsers?

**Reply by zandbelt** [reply]
no, no plans either: it is practically impossible to guarantee (poor man's solutions set aside)

**Reply by zaic20** [reply]
Thanks @zandbelt, much appreciated!!!
